charts/tefca-viewer/templates/deployment.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: "{{.Release.Name}}-{{.Values.appName}}"
+  labels:
+    app: {{.Values.appName}}
+spec:
+  replicas: {{.Values.replicaCount}}
+  selector:
+    matchLabels:
+      app: {{.Values.appName}}
+  template:
+    metadata:
+      annotations:
+      labels:
+        app: {{.Values.appName}}
+    spec:
+      imagePullSecrets:
+      containers:
+        - name: {{.Chart.Name}}
+          image: ghcr.io/cdcgov/phdi/{{.Values.name}}:{{ .Values.image.tag }}
+          imagePullPolicy: {{.Values.image.pullPolicy}}
+          ports:
+            - name: {{.Chart.Name | trunc 15}}
```

Comment:
Would double-check this truncation to 15 - Azure limits your chart name length, and we'd run into issues previously with the chart name terminating on a bad character. (like `orchestration-` - Azure can't handle the dash at the end) I think you're probably fine here but may want to double-check.

Reply:
good catch! I double checked and it will truncate to be `tefca-viewer-cha`. Awkward but not a bad character termination.